From: Georg Faerber <georg@debian.org>
Date: Sat, 12 Jun 2021 12:41:03 +0900
Subject: tests: skip failing

Last-Update: 2020-02-04

Some tests do fail, as they assume the vendored version of libxml inside
nokogiri. We do rely on system libraries, which is why this fails for us.

See https://github.com/rails/rails-html-sanitizer/issues/99 for details.
Last-Update: 2020-02-04
---
 test/sanitizer_test.rb | 4 ++++
 1 file changed, 4 insertions(+)

Index: ruby-rails-html-sanitizer/test/sanitizer_test.rb
===================================================================
--- ruby-rails-html-sanitizer.orig/test/sanitizer_test.rb
+++ ruby-rails-html-sanitizer/test/sanitizer_test.rb
@@ -482,6 +482,7 @@ class SanitizersTest < Minitest::Test
   end
 
   def test_uri_escaping_of_href_attr_in_a_tag_in_safe_list_sanitizer
+    skip
     skip if RUBY_VERSION < "2.3"
 
     html = %{<a href='examp<!--" unsafeattr=foo()>-->le.com'>test</a>}
@@ -492,6 +493,7 @@ class SanitizersTest < Minitest::Test
   end
 
   def test_uri_escaping_of_src_attr_in_a_tag_in_safe_list_sanitizer
+    skip
     skip if RUBY_VERSION < "2.3"
 
     html = %{<a src='examp<!--" unsafeattr=foo()>-->le.com'>test</a>}
@@ -502,6 +504,7 @@ class SanitizersTest < Minitest::Test
   end
 
   def test_uri_escaping_of_name_attr_in_a_tag_in_safe_list_sanitizer
+    skip
     skip if RUBY_VERSION < "2.3"
 
     html = %{<a name='examp<!--" unsafeattr=foo()>-->le.com'>test</a>}
@@ -512,6 +515,7 @@ class SanitizersTest < Minitest::Test
   end
 
   def test_uri_escaping_of_name_action_in_a_tag_in_safe_list_sanitizer
+    skip
     skip if RUBY_VERSION < "2.3"
 
     html = %{<a action='examp<!--" unsafeattr=foo()>-->le.com'>test</a>}
